package-lock.json

```diff
@@ -3352,6 +3352,16 @@
       "resolved": "https://registry.npmjs.org/@types/history/-/history-4.7.3.tgz",
       "integrity": "sha512-cS5owqtwzLN5kY+l+KgKdRJ/Cee8tlmQoGQuIE9tWnSmS3JMKzmxo2HIAk2wODMifGwO20d62xZQLYz+RLfXmw=="
     },
+    "@types/hoist-non-react-statics": {
```

Comment:
If there's no new packages, you don't need to submit changes to this file. Can you revert changes to this file?

Reply:
Alright, thanks for the heads-up!